package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@openinf/util-text",
   "version": "1.1.2",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
This is not desirable &mdash; we will be switching package managers soon, so these lockfiles will be different anyways&hellip;